requirements.txt

```diff
@@ -3,9 +3,9 @@ tqdm
 pandas
 numpy
 opencv-python
-Pillow==9.4.0
-onnxruntime # onnxruntime-gpu for GPU support
-librosa==0.9.2
+Pillow>=9.4.0
+onnxruntime>=1.15.0  # onnxruntime-gpu for GPU support
+librosa>=0.9.2
 matplotlib
-onnx==1.14.0
-tf2onnx==1.14.0
+onnx>=1.14.0
+tf2onnx>=1.14.0
```

Comment:
I think better to use == instead of >=, because in the future libraries might be updated in a way that my library may not work with newer versions and there could be many problems, that's why it would be great to have some unittests so I could tests if everything works the same with never versions

Reply:
What you're suggesting is definitely a good practice for a program but in the case of a library, used by others, constraining the dependent libraries to a specific version creates issues when the library is used in conjunction with other libraries, because it can often create unreconcilable conflicts in required versions.